lib/modules.nix

```diff
@@ -87,6 +84,16 @@ rec {
       result = { inherit options config; };
     in result;
 
+
+ /* Filter disabled modules. Modules can be disabled allowing
+    their implementation to be replaced. */
+ filterModules = modules:
+   let
+     moduleKey = m: if isString m then toString ../nixos/modules + "/" + m else toString m;
```

Comment:
That's why I left in the option to use full paths, or are  you referring to the `nixos/modules` prefix?